packages/dear-imgui/cpp/app/src/main.cpp

```
@@ -164,100 +164,100 @@ class WasmRunner {
 
         void appendChild(const int parentId, const int childId) const {
-            m_view->QueueAppendChild(parentId, childId);
+            m_reactImgui->QueueAppendChild(parentId, childId);
         }
 
         [[nodiscard]] std::vector<int> getChildren(const int id) const {
-            return m_view->GetChildren(id);
+            return m_reactImgui->GetChildren(id);
         }
 
         [[nodiscard]] std::string getAvailableFonts() const {
```

cppcheck

[style] The function 'getAvailableFonts' is never used.

(unusedFunction)